bin/MadGraph5_aMCatNLO/cards/production/13TeV/exo_diboson/Spin-2/RSGraviton_hh_hVVhbb_fullLep/RSGraviton_hh_hVVhbb_fullLep_narrow_M1000/RSGraviton_hh_hVVhbb_fullLep_narrow_M1000_proc_card.dat

```diff
@@ -0,0 +1,10 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set gauge unitary
+set complex_mass_scheme False
+import model RS_exovv
+define lep = e+ e- mu+ mu-
```

Comment:
We have checked with analyzers. They explicitly say no, they don't want tau.

(while all other ExoDiboson channels include tau in the final state)